packages/aquila-stylelint/package.json

```diff
@@ -20,12 +20,10 @@
     },
     "homepage": "https://github.com/AquilaLearning/aquila-linting#readme",
     "peerDependencies": {
+        "@stylistic/stylelint-config": "2.x",
         "stylelint": "16.x",
         "stylelint-no-unsupported-browser-features": "8.x",
         "stylelint-order": "6.x",
         "stylelint-scss": "6.x"
-    },
-    "devDependencies": {
-        "@stylistic/stylelint-config": "^2.0.0"
```

Comment:
Surely it still needs to be a dev dependency?  It won't install peer dependencies?

Reply:
Anything that this package requires to operate should be specified as a peer dependency.  I.e. anything we're specifying here https://github.com/AquilaLearning/aquila-linting/blob/main/packages/aquila-stylelint/README.md?plain=1#L14

Which is a point actually we need to add stylistic to this.

My point was more that in order to test this config within this repo we also need to install these, so they should also be devDependencies.

Reply:
As you mentioned, seems like we can install the peer dependency:

>UPDATE:
npm versions 1, 2, and 7 will automatically install peerDependencies if they are not explicitly depended upon higher in the dependency tree. For npm versions 3 through 6, you will receive a warning that the peerDependency is not installed instead.

https://nodejs.org/en/blog/npm/peer-dependencies